.pre-commit-config.yaml

```diff
@@ -144,11 +144,11 @@ repos:
 
 # Check for spelling
 - repo: https://github.com/codespell-project/codespell
-  rev: "v2.1.0"
+  rev: "v2.2.1"
   hooks:
   - id: codespell
     exclude: ".supp$"
-    args: ["-L", "nd,ot,thist"]
+    args: ["-L", "nd,ot,thist,intstruct,ro"]
```

Comment:
JIC you find it useful, on the smart_holder branch we have this:

* https://github.com/pybind/pybind11/blob/6df86934c258d8cd99acf192f6d3f4d1289b5d68/.pre-commit-config.yaml#L154
* https://github.com/pybind/pybind11/blob/6df86934c258d8cd99acf192f6d3f4d1289b5d68/.codespell-ignorelines

The main advantage is that the suppressions are more targeted: entire lines have to match.

Additional tiny advantage only for me: if we introduce the .codespell-ignorelines file here on master it'll be slightly easier for me on the smart_holder side.

Reply:
@rwgk Let's introduce the file.